### PR TITLE
Updated pre-need form with schema changes and fixes for chapter 1.

### PR DESCRIPTION
**Pull request by U-DON**
Resolves department-of-veterans-affairs/vets.gov-team#5411.
Resolves department-of-veterans-affairs/vets.gov-team#5130.

Schema changes are mostly updated validations, types, and enum options.

### CODE REVIEW

File: src/js/pre-need/config/form.js [hunk 1]
```diff
@@ -170,7 +182,9 @@ const formConfig = {
                         'placeOfBirth',
                         'maritalStatus',
                         'militaryStatus'
-                      ], veteran.properties)
+                      ], _.set('militaryStatus.enum', [
+                        'V', 'R', 'A', 'E', 'S', 'O', 'X'
```
Comment (U-DON) [inline src/js/pre-need/config/form.js]: This is to remove statuses involving death from the options that a presumably living veteran is filling out for themselves.

File: src/js/pre-need/config/form.js [hunk 2]
```diff
@@ -84,32 +88,35 @@ const formConfig = {
...
                   },
+                  /*
+                  'view:other': {
+                    'ui:title': 'Please specify',
+                    'ui:required': (form) => get(form, 'relationship.type') === '4',
+                    'ui:options': {
+                      hideIf: (form) => get(form, 'relationship.type') !== '4'
+                    }
+                  }
+                  */
```
Comment (U-DON) [inline src/js/pre-need/config/form.js]: Doesn't map to anything in schema, so should probably be removed, but will be clarified with EOAS.

File: src/js/pre-need/config/schema.json [hunk 3]
```diff
@@ -198,23 +198,22 @@
         "street": {
           "type": "string",
           "minLength": 1,
-          "maxLength": 50
+          "maxLength": 35
```
Comment (U-DON) [inline src/js/pre-need/config/schema.json]: These are all copied over from the built JSON schema from `vets-json-schema` as usual for development.

**Review by bshyong — APPROVED** [review]
LGTM

**Pull request merged**